recipes/fire-hpp/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class FireHppConan(ConanFile):
+    name = "fire-hpp"
+    homepage = "https://github.com/kongaskristjan/fire-hpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Fire for C++: Create fully functional CLIs using function signatures"
+    topics = ("command-line", "argument", "parser")
+    license = "BSL-1.0"
+    no_copy_source = True
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = ["bsl_1_0"]
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    def source(self):
```

Comment:
I think we need a test for `c++11` in configure: https://github.com/kongaskristjan/fire-hpp/blob/760ff08ebe5ebfc09db34972cc759356c40bf52c/CMakeLists.txt#L30
```python
def configure(self):
    if self.settings.compiler.cppstd:
        tools.check_min_cppstd(self, 11)
```

Reply:
Agreed.